lib/library/library_model.dart

```diff
@@ -308,4 +309,10 @@ class LibraryModel extends SafeChangeNotifier implements NavigatorObserver {
   final GlobalKey<NavigatorState> _masterNavigatorKey =
       GlobalKey<NavigatorState>();
   GlobalKey<NavigatorState> get masterNavigatorKey => _masterNavigatorKey;
+
+  AudioType getCurrentAudioType() => switch (selectedPageId) {
```

Comment:
this sadly doesnt always work, as starred stations and subbed podcasts also are put into the sidebar and can be selected :(

Reply:
but you could add a function which uses something like isPageInLibrary like checking playlists, podcasts, starredStations, pinnedAlbums and the main page ids